NitroxServer/ConsoleCommands/Abstract/Command.cs

```diff
@@ -65,9 +73,18 @@ public void TryExecute(Optional<Player> sender, string[] args)
             }
         }
 
+        public bool CanExecute(Perms treshold)
+        {
+#if RELEASE
+            return RequiredPermLevel <= treshold && !Flags.HasFlag(PermsFlag.DEBUG_ONLY);
+#else
+            return RequiredPermLevel <= treshold;
+#endif
+        }
```

Comment:
Omit preprocessor directives here, omit "debug" commands instead if in release mode.

```suggestion
        public bool CanExecute(Perms threshold)
        {
            return RequiredPermLevel <= threshold;
        }
```